src/ml_board/frontend/Dashboard/src/app/App.tsx

```diff
@@ -115,34 +115,17 @@ export default function App() {
 
     // TODO: maybe useCallback
     const workerOnMessageHandler = (data: DataToRedux) => {
-        if (data && data.evaluationResultsData) {
+        //  abstract equality operator, so null == undefined
+        if (data == undefined) return;
```

Comment:
shouldn't this be handled one level above (the function calling `workerOnMessageHandler`)? If there was a message coming in that is undefined, I would assume that must be an error?